Documentation update (#26)
gini/gini-pay-bank-sdk-ios: Documentation/source/Integration.md

```swift
bankSDK.returnBackToBusinessAppHandler(resolvedPaymentRequest: resolvedPayment)
```
## Gini Pay Scheme For Your App

In order for banking apps to be availible as a payment provider and support Gini Pay Connect functionality you need register a scheme for your app known by the Gini Pay API.

You should already have a scheme and host from us. Please contact us in case you don't have them.

The following is an example for the scheme ginipay-bank://
<br>
<center><img src="img/Integration guide/BankSchemeExample.png" height="200"/></center>
</br>

## Testing

An example business app is available in the [Gini Pay Business SDK's](https://github.com/gini/gini-pay-business-sdk-ios) repository.

In order to test using our example business app you need to use development client credentials. This will make sure
the Gini Pay Business SDK uses a test payment provider which will open our example banking app.

To inject your API credentials into the banking example app you need to fill in your credentials in `Example/Bank/Credentials.plist`.

> 

Reviewer: ```suggestion
You also need to replace the `ginipay-bank` URL scheme in the example business app's `LSApplicationQueriesSchemes` in the `Info.plist` with the Gini Pay URL scheme we provided for your banking app.

```

#### End to end testing

The app scheme in our business example app: `ginipay-business://`.
After you've set the client credentials in the example business app and installed it on your device you can start the payment flow with a document import or make a photo.

After following the integration steps above you'll be redirected back to the business app.

With these steps completed you have verified that your app, the Gini Pay API, the Gini Pay Business SDK and the Gini Pay
Bank SDK work together correctly.

#### Testing in production

The steps are the same but instead of the development client credentials you will need to use production client
credentials. This will make sure the Gini Pay Business SDK receives real payment providers which open real banking apps.

> Make sure that for production you register the scheme we provided you and you are not using `ginipay-bank://`.

Reviewer: This line is not needed since they will register the correct scheme we give them during integration. The Gini Pay API will return their scheme in the payment provider for their client credentials. They don't need to use the `ginipay-bank://` scheme in their banking app.

> For testing the whole flow you will need to specify the production scheme `LSApplicationQueriesSchemes` in `Info.plist` file in the business app which uses the Gini Pay Business SDK and install it.

Reviewer: ```suggestion
For testing the flow using the example business app you will need to add your banking app's production Gini Pay URL scheme to `LSApplicationQueriesSchemes` in the example business app's `Info.plist`. Also please make sure that production client credentials are used before installing it.
```

> Please contact us in case you don't know which business app(s) to install for starting the payment flow.

Reviewer: ```suggestion
You can also test with a real business app. Please contact us in case you don't know which business app(s) to install for starting the payment flow.
```
